cmake/InstallDirs.cmake

```diff
@@ -1,3 +1,19 @@
+# We used to override CMAKE_INSTALL_LIBDIR to always default to "lib", but starting with 5.0, we now
+# use the distribution-defaults for library installation directories. Since CMAKE_INSTALL_LIBDIR is
+# a cache variable, let's try and detect if an old default is in the cache.
+function(warn_for_old_libdir_cache)
+  if(CMAKE_INSTALL_LIBDIR STREQUAL "lib"
+     AND EXISTS "${CMAKE_BINARY_DIR}/CMakeCache.txt"
+     AND NOT IRODS_INSTALL_LIBDIR_GOOD_DEFAULT)
```

Comment:
So `IRODS_INSTALL_LIBDIR_GOOD_DEFAULT` is a way to tell the build to skip this check?

Reply:
Correct.